.pre-commit-config.yaml

```diff
@@ -58,7 +58,7 @@ repos:
     hooks:
       - id: mypy
         additional_dependencies: ["types-PyYAML", "types-requests"]
-        exclude: ^(docs/|.*/test_.*|utilities/manifests/*|utilities/plugins/tgis_grpc/*)
+        exclude: ^(docs/|.*test.*\.py$|.*/tests/.*\.py$|utilities/manifests/.*|utilities/plugins/tgis_grpc/.*)
```

Comment:
`.*/tests/.*\.py` this would exclude conftest.py and utils.py under tests. We don't want that.